Sources/SWBCore/SpecImplementations/Tools/SwiftCompiler.swift

```diff
@@ -2161,11 +2160,11 @@ public final class SwiftCompilerSpec : CompilerSpec, SpecIdentifierType, SwiftDi
 
             // Add const metadata outputs to extra compilation outputs
             if await supportConstSupplementaryMetadata(cbc, delegate, compilationMode: compilationMode) {
-                // If using whole module optimization then we use the -master.swiftconstvalues file from the sole compilation task.
```

Comment:
Does primary make the most sense here or is there another word which would be even clearer? Like, what is this the primary of? Or is it more of an aggregation?

Reply:
It refers to the primary entry in the output file map, which I think is accurate